meta-avocado/recipes-avocado/extensions/avocado-extension.bb

```diff
@@ -0,0 +1,87 @@
+SUMMARY = "Avocado combination system/configuration extension"
```

Comment:
I think we should make this file .inc or a class as to not make it seem like it can be built as a stand alone recipe.

Reply:
I agree.